app/schema.graphql

```diff
@@ -1472,6 +1478,18 @@ type PromptVersionEdge {
   node: PromptVersion!
 }
 
+input PromptVersionInput {
+  description: String = null
+  templateType: PromptTemplateType!
+  templateFormat: PromptTemplateFormat!
+  template: JSON!
```

Comment:
This is the weird part of our schema - this actually determines the template type. I guess this is the most flexible for now so I'm good with it but I think it's probably worth making this a strict type

Reply:
As it will force code changes when the template formats change